autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -102,7 +103,12 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
         hyperparameter_tune : (bool, default = False)
             Whether to tune hyperparameters or just use fixed hyperparameter values for each model. Setting as True will increase `fit()` runtimes.
         feature_prune : (bool, default = False)
-            Whether or not to perform feature selection. 
+            Whether or not to perform feature selection.
+        auto_stack : (bool, default = False)
+            Whether to have AutoGluon automatically attempt to select optimal num_bagging_folds and stack_ensemble_levels based on data properties.
+            Note: Overrides num_bagging_folds and stack_ensemble_levels values.
```

Comment:
Assuming we keep `auto_stack` as is, does this override num_bagging_sets as well?  
It seems like this value should also be automatically set by `auto_stack`

Reply:
num_bagging_sets is extremely time consuming, which is why it isn't controlled by auto_stack currently. It is set to 1 by default and 10 by default if time_limits is set.

Reply:
The one argument for setting it is to set it above 1 when the data has few rows to help mitigate overfitting, but I think we should save that optimization for later, since this requires a more thorough investigation into how num_bagging_sets improves results